src/helics/apps/AsioBrokerServer.cpp

```diff
@@ -94,7 +94,7 @@ namespace udp {
 
         asio::ip::udp::socket mSocket;
         asio::ip::udp::endpoint mRemoteEndpoint;
-        std::array<char, 1024> mRecvBuffer{0};
+        std::array<char, 1024> mRecvBuffer{{0}};
```

Comment:
Is this something like nested value initialization, with the inner `{0}` specifying the value for the individual char entries, and `{{0}}` setting the initial value for the entire array to 0's?

Reply:
one of the clang compilers gave a warning on this, something about partial initializer lists and recommended this fix.

Reply:
and no it is just the first value that is set to 0